workflow/controller/operator.go

```diff
@@ -132,6 +142,7 @@ func newWorkflowOperationCtx(wf *wfv1.Workflow, wfc *WorkflowController) *wfOper
 		deadline:               time.Now().UTC().Add(maxOperationTime),
 		auditLogger:            argo.NewAuditLogger(wf.ObjectMeta.Namespace, wfc.kubeclientset, wf.ObjectMeta.Name),
 		preExecutionNodePhases: make(map[string]wfv1.NodePhase),
+		submissionParameters:   wf.Spec.DeepCopy().Arguments.Parameters,
```

Comment:
Should this be
```suggestion
		submissionParameters:   wf.Spec.Arguments.Parameters.DeepCopy(),
```

Instead?